### PR TITLE
fix design on addressbook add page

### LINKED ISSUE #3425: Layout issue : Address book
Maybe, "add a new entry" page in the address book should be similar to "sweep paper wallet" page, where scan symbol is in the label and also a placeholder is show.

d015b9ce-9f33-45cc-81fc-ef5e263699bb

alrbk0zjirsa1vmnq7e2z_ts0gehqduipe0pc2wix9au

### PR DESCRIPTION
**Pull request by tanojaja**
Fixes #3425

### CODE REVIEW

File: public/views/modals/destination-address.html [hunk 1]
```diff
@@ -104,9 +102,15 @@ <h4 class="title m0" translate>Add a new entry</h4>
...
+          <div class="input">
+            <input type="text" id="address" name="address" ng-model="addressbook.address" valid-address required placeholder="Put the destination address here">
+            <div class="qr-scanner-input">
+              <qr-scanner ng-show="addAddressbookEntry" on-scan="onQrCodeScanned(data, addressbookForm)" before-scan="beforeQrCodeScann()"></qr-scanner>
+            </div>
+          </div>
+          
           <label translate>Label</label>
-          <input type="text" id="label" name="label" ng-model="addressbook.label" required>
+          <input type="text" id="label" name="label" ng-model="addressbook.label" required placeholder="Enter the wallet name">
```
Comment (cmgustavo) [inline public/views/modals/destination-address.html]: Why "the wallet name" ?

Comment (tanojaja) [inline public/views/modals/destination-address.html]: It should be address name, right?

Comment (cmgustavo) [inline public/views/modals/destination-address.html]: It 's just a label

**Comment by cmgustavo** [comment]
Please, add the attribute `translate` to the new strings

**Comment by tanojaja** [comment]
Updated

**Comment by tanojaja** [comment]
updated

**Comment by tanojaja** [comment]
Updated.

**Comment by cmgustavo** [comment]
LGTM

**Pull request merged**